tajo-core/src/main/proto/InternalTypes.proto

```diff
@@ -45,3 +45,8 @@ message CorrProto {
   required double yvar = 5;
   required double covar = 6;
 }
+
+message StringAggProto {
+  required string value = 1;
```

Comment:
It needs to keep the second parameter, delimiter, in the merge method of StringAgg  when the second stages execute to merge the result of fisrt stage of StringAgg.